README.md

```
@@ -428,6 +446,40 @@ When making API requests, include the token in the Authorization header:
 $ curl -H "Authorization: Bearer YOUR_TOKEN" http://localhost:3000/zones
 ```
 
+### Prometheus Metrics
+
+When `api.metrics_enabled` is on (the default), bindizr serves Prometheus text-format metrics at `GET /metrics`. Like `/health`, the endpoint is unauthenticated — it exposes only aggregate counters and gauges, never zone data — and it is not part of the OpenAPI spec.
+
+```bash
+$ curl http://localhost:3000/metrics
```

markdownlint-cli2 (0.23.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)